### PR TITLE
[17.0][MIG] base_import_pdf_by_template: Migration to 17.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 17.0

Please @CarlosRoca13 and @pilarvargas-tecnativa can you review it?

@Tecnativa TT50003

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
:+1: Code review

**Review by pilarvargas-tecnativa — APPROVED** [review]
:+1:

**Comment by victoralmau** [comment]
Ping @pedrobaeza

File: base_import_pdf_by_template/readme/USAGE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 This module allows to upload PDF files in any Odoo model. It processes
 each of the files and converts it into a new record. Technically, the
 pdf is transformed into text and that text is processed to create the
-record. The module incorporates an option in Favorites Import PDF and
+record. The module incorporates an option in actions 'Import PDF and
```
Comment (pedrobaeza) [inline base_import_pdf_by_template/readme/USAGE.md]: Isn't missing the final `'` ?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**